maintenance_plan/security/maintenance_plan_security.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="maintenance_plan_comp_rule" model="ir.rule">
+            <field name="name">Maintenance Plan Multi-company rule</field>
+            <field name="model_id" ref="model_maintenance_plan"/>
+            <field name="global" eval="True"/>
+            <field name="domain_force">
+                [
+                    '|',
+                    ('equipment_id.company_id','=',False),
```

Comment:
Just one thing, in order to do this kind of search, you should check that the field equipment_id has auto_join

Reply:
@etobella looking at `equipment_id` definition in `maintenance.plan` model, `auto_join` is not defined. I'll add it ASAP.

Reply:
> @etobella looking at equipment_id definition in maintenance.plan model, auto_join is not defined. I'll add it ASAP.

Done!